<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UmsConfiguration</class>
 <widget class="QWidget" name="UmsConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB Mass Storage device configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_collectionName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_autoConnect">
     <property name="text">
      <string>Use automatically when connected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_musicCheckBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="Enabled music storage on this device"/>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="m_musicWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Music folder:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KUrlRequester" name="m_musicFolder"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="m_filenameSchemeBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Filename scheme of tracks that are added to the device</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_transcodeLabel">
          <property name="text">
           <string>Transcode:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Transcoding::SelectConfigWidget" name="m_transcodeConfig"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_podcastCheckBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="Enabled podcast storage on this device"/>
     </property>
     <property name="text">
      <string>Podcasts</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="m_podcastWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Podcast folder:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KUrlRequester" name="m_podcastFolder"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Transcoding::SelectConfigWidget</class>
   <extends>QComboBox</extends>
   <header>transcoding/TranscodingSelectConfigWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_musicCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_musicWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_podcastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_podcastWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
